### ISSUE TITLE
`PasteKnownSourceIntoSingleLineRawInterpolatedStringTests` are failing locally

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#62880]
```
[xUnit.net 00:01:34.32]     Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.PasteKnownSourceIntoSingleLineRawInterpolatedStringTests.TestPasteSimpleRawMultiLineLiteralContent2 [FAIL]
[xUnit.net 00:01:34.32]       Assert.True() Failure
[xUnit.net 00:01:34.32]       Expected: True
[xUnit.net 00:01:34.32]       Actual:   False
[xUnit.net 00:01:34.32]       Stack Trace:
[xUnit.net 00:01:34.32]         src\EditorFeatures\CSharpTest\StringCopyPaste\StringCopyPasteCommandHandlerTests.cs(106,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.StringCopyPasteCommandHandlerTests.StringCopyPasteTestState.<>c__DisplayClass7_1.<TestCopyPaste>b__3()
[xUnit.net 00:01:34.32]         src\EditorFeatures\CSharp\StringCopyPaste\StringCopyPasteCommandHandler_CutCopy.cs(53,0): at Microsoft.CodeAnalysis.Editor.CSharp.StringCopyPaste.StringCopyPasteCommandHandler.ExecuteCutOrCopyCommand(ITextView textView, ITextBuffer subjectBuffer, Action nextCommandHandler, CommandExecutionContext executionContext)
[xUnit.net 00:01:34.32]         src\EditorFeatures\CSharp\StringCopyPaste\StringCopyPasteCommandHandler_CutCopy.cs(45,0): at Microsoft.CodeAnalysis.Editor.CSharp.StringCopyPaste.StringCopyPasteCommandHandler.ExecuteCommand(CopyCommandArgs args, Action nextCommandHandler, CommandExecutionContext executionContext)
[xUnit.net 00:01:34.32]         src\EditorFeatures\CSharpTest\StringCopyPaste\StringCopyPasteCommandHandlerTests.cs(102,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.StringCopyPasteCommandHandlerTests.StringCopyPasteTestState.TestCopyPaste(String expectedMarkup, String pasteText, Boolean pasteTextIsKnown, String afterUndoMarkup)
[xUnit.net 00:01:34.32]         src\EditorFeatures\CSharpTest\StringCopyPaste\StringCopyPasteCommandHandlerKnownSourceTests.cs(21,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.StringCopyPasteCommandHandlerKnownSourceTests.TestCopyPaste(String copyFileMarkup, String pasteFileMarkup, String expectedMarkup, String afterUndo, Boolean mockCopyPasteService)
[xUnit.net 00:01:34.32]         src\EditorFeatures\CSharpTest\StringCopyPaste\PasteKnownSourceIntoSingleLineRawInterpolatedStringTests.cs(661,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.PasteKnownSourceIntoSingleLineRawInterpolatedStringTests.TestPasteSimpleRawMultiLineLiteralContent2()
[xUnit.net 00:01:36.58]     Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.PasteKnownSourceIntoSingleLineRawInterpolatedStringTests.TestPasteQuotesRawSingleLineLiteralContent [FAIL]
[xUnit.net 00:01:36.58]       Assert.True() Failure
[xUnit.net 00:01:36.58]       Expected: True
[xUnit.net 00:01:36.58]       Actual:   False
[xUnit.net 00:01:36.58]       Stack Trace:
[xUnit.net 00:01:36.58]         src\EditorFeatures\CSharpTest\StringCopyPaste\StringCopyPasteCommandHandlerTests.cs(106,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.StringCopyPasteCommandHandlerTests.StringCopyPasteTestState.<>c__DisplayClass7_1.<TestCopyPaste>b__3()
[xUnit.net 00:01:36.58]         src\EditorFeatures\CSharp\StringCopyPaste\StringCopyPasteCommandHandler_CutCopy.cs(53,0): at Microsoft.CodeAnalysis.Editor.CSharp.StringCopyPaste.StringCopyPasteCommandHandler.ExecuteCutOrCopyCommand(ITextView textView, ITextBuffer subjectBuffer, Action nextCommandHandler, CommandExecutionContext executionContext)
[xUnit.net 00:01:36.58]         src\EditorFeatures\CSharp\StringCopyPaste\StringCopyPasteCommandHandler_CutCopy.cs(45,0): at Microsoft.CodeAnalysis.Editor.CSharp.StringCopyPaste.StringCopyPasteCommandHandler.ExecuteCommand(CopyCommandArgs args, Action nextCommandHandler, CommandExecutionContext executionContext)
[xUnit.net 00:01:36.58]         src\EditorFeatures\CSharpTest\StringCopyPaste\StringCopyPasteCommandHandlerTests.cs(102,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.StringCopyPasteCommandHandlerTests.StringCopyPasteTestState.TestCopyPaste(String expectedMarkup, String pasteText, Boolean pasteTextIsKnown, String afterUndoMarkup)
[xUnit.net 00:01:36.58]         src\EditorFeatures\CSharpTest\StringCopyPaste\StringCopyPasteCommandHandlerKnownSourceTests.cs(21,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.StringCopyPasteCommandHandlerKnownSourceTests.TestCopyPaste(String copyFileMarkup, String pasteFileMarkup, String expectedMarkup, String afterUndo, Boolean mockCopyPasteService)
[xUnit.net 00:01:36.58]         src\EditorFeatures\CSharpTest\StringCopyPaste\PasteKnownSourceIntoSingleLineRawInterpolatedStringTests.cs(577,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.PasteKnownSourceIntoSingleLineRawInterpolatedStringTests.TestPasteQuotesRawSingleLineLiteralContent()
[xUnit.net 00:01:39.93]     Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.PasteKnownSourceIntoSingleLineRawInterpolatedStringTests.TestPasteSimpleRawMultiLineLiteralContent3B [FAIL]
[xUnit.net 00:01:39.93]       Assert.True() Failure
[xUnit.net 00:01:39.93]       Expected: True
[xUnit.net 00:01:39.93]       Actual:   False
[xUnit.net 00:01:39.93]       Stack Trace:
[xUnit.net 00:01:39.93]         src\EditorFeatures\CSharpTest\StringCopyPaste\StringCopyPasteCommandHandlerTests.cs(106,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.StringCopyPasteCommandHandlerTests.StringCopyPasteTestState.<>c__DisplayClass7_1.<TestCopyPaste>b__3()
[xUnit.net 00:01:39.93]         src\EditorFeatures\CSharp\StringCopyPaste\StringCopyPasteCommandHandler_CutCopy.cs(53,0): at Microsoft.CodeAnalysis.Editor.CSharp.StringCopyPaste.StringCopyPasteCommandHandler.ExecuteCutOrCopyCommand(ITextView textView, ITextBuffer subjectBuffer, Action nextCommandHandler, CommandExecutionContext executionContext)
[xUnit.net 00:01:39.93]         src\EditorFeatures\CSharp\StringCopyPaste\StringCopyPasteCommandHandler_CutCopy.cs(45,0): at Microsoft.CodeAnalysis.Editor.CSharp.StringCopyPaste.StringCopyPasteCommandHandler.ExecuteCommand(CopyCommandArgs args, Action nextCommandHandler, CommandExecutionContext executionContext)
[xUnit.net 00:01:39.93]         src\EditorFeatures\CSharpTest\StringCopyPaste\StringCopyPasteCommandHandlerTests.cs(102,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.StringCopyPasteCommandHandlerTests.StringCopyPasteTestState.TestCopyPaste(String expectedMarkup, String pasteText, Boolean pasteTextIsKnown, String afterUndoMarkup)
[xUnit.net 00:01:39.93]         src\EditorFeatures\CSharpTest\StringCopyPaste\StringCopyPasteCommandHandlerKnownSourceTests.cs(21,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.StringCopyPasteCommandHandlerKnownSourceTests.TestCopyPaste(String copyFileMarkup, String pasteFileMarkup, String expectedMarkup, String afterUndo, Boolean mockCopyPasteService)
[xUnit.net 00:01:39.93]         src\EditorFeatures\CSharpTest\StringCopyPaste\PasteKnownSourceIntoSingleLineRawInterpolatedStringTests.cs(745,0): at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.StringCopyPaste.PasteKnownSourceIntoSingleLineRawInterpolatedStringTests.TestPasteSimpleRawMultiLineLiteralContent3B()
```

@CyrusNajmabadi Can you take a look please?

All what I did to reproduce was right clicking `Microsoft.CodeAnalysis.CSharp.EditorFeatures.UnitTests` in solution explorer and hit "Run tests"

**Comment by davidwengier** [comment]
Were you doing anything else on the machine at the time? Not sure if its exactly these tests, but some of them interact with the clipboard on the machine, so can be sensetive to that.

**Comment by Youssef1313** [comment]
@davidwengier Thanks. I was messing with clipboard indeed. However, I still see (flaky) failures. I ran the tests few times without interacting with clipboard, sometimes they all pass, sometimes I see 2 or 3 failures.

**Comment by CyrusNajmabadi** [comment]
Were you interacting with the machine at all?  These tests will not work if you do that as they are issuing actual copy commands through the editor itself.

**Comment by Youssef1313** [comment]
@CyrusNajmabadi I ran it now without interacting with anything and 5 tests failed.

**Comment by Youssef1313** [comment]
Do VS run the tests in parallel? I have no idea about that, but it would make sense if they're run in parallel.

**Comment by CyrusNajmabadi** [comment]
I've tried this locally without any issue.

**Comment by Youssef1313** [comment]
I don't know :(
Anyway I think it's reasonable to close this as a duplicate of https://github.com/dotnet/roslyn/issues/61764
